lib/src/_shared/PickerPopper.tsx

```diff
@@ -49,7 +49,7 @@ export const useStyles = makeStyles(
       transformOrigin: 'bottom center',
     },
   }),
-  { name: 'MuiPickersPopper' }
+  { name: 'MuiPickerPopper' }
```

Comment:
```suggestion
  { name: 'MuiPickersPopper' }
```

Per the conclusion of #1778